test/unit/accept_patient_reports.js

```diff
@@ -17,6 +17,9 @@ exports.tearDown = function(callback) {
     if (utils.getRegistrations.restore) {
         utils.getRegistrations.restore();
     }
+    if (utils.getPatientContactUuid.restore) {
+        utils.getPatientContactUuid.restore();
+    }
```

Comment:
Heh, test_utils happened after this change. I have a note to go through *all of sentinel* after this has been approved and change everything. I'll change this one in this PR and then do all the rest in a separate one.